src/main/webapp/dev/js/common/bootboxWrapper.es6

```diff
@@ -32,8 +32,12 @@ function showModalAlert(titleText, messageText, okButtonText, color) {
             },
         },
     })
-    // applies bootstrap color to title background
-    .find('.modal-header').addClass(`alert-${color}` || StatusType.DEFAULT);
+    .find('.modal-header')
+        // applies bootstrap color to title background
+        .addClass(`alert-${color}` || StatusType.DEFAULT)
```

Comment:
Caught a problem with the original code: instead of
```javascript
.addClass(`alert-${color}` || StatusType.DEFAULT)
```
it should be
```javascript
.addClass(`alert-${color || StatusType.DEFAULT}`)
```

Even though `color` is supposed to be optional, calling the function now without supplying it will result in an `alert-undefined` class being added instead of `alert-info`.

Might as well fix it here since you'll be changing all the affected lines :)